packages/limesuite/PKGBUILD

```diff
@@ -0,0 +1,46 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=limesuite
+pkgver=23.11.0.r19.g524cd2e
+pkgrel=1
+pkgdesc='Driver and GUI for LMS7002M-based SDR platforms.'
+arch=('x86_64' 'aarch64')
+groups=('blackarch' 'blackarch-radio')
+url='https://github.com/myriadrf/LimeSuite'
+license=('Apache-2.0')
+depends=('freeglut' 'i2c-tools' 'libusb' 'soapysdr' 'sqlite' 'wxwidgets-gtk3')
```

Comment:
Building with ba-dev fails

<img width="1846" height="684" alt="Image" src="https://github.com/user-attachments/assets/c1f511dd-ebf9-48b6-8882-8c139d694f85" />

It seems OpenGL is missing, so `mesa` should be added as a dependency. See https://wiki.archlinux.org/title/OpenGL.

Reply:
Sorry, I was wrong, mesa was not needed, it was `glu` that was needed in addition to `freeglut`. Else `usr/include/GL/glu.h` is missing and `OpenGL::GLU` is not satisfied. Please also test if `freeglut` is needed at all.